decidim-core/app/packs/src/decidim/redesigned_index.js

```diff
@@ -43,40 +40,52 @@ import "./append_redirect_url_to_modals"
 import "./form_attachments"
 import "./form_remote"
 // import "./conferences" -- deprecated
```

Comment:
This is a way to track which scripts become legacy... once the renaming is done, then we can remove these files. Now are being used in decidim-core/app/packs/src/decidim/index.js.

Since the renaming is another branch #11123, this fits better there